client/modules/User/actions.js

```diff
@@ -59,6 +59,11 @@ export function validateAndLoginUser(previousPath, formProps, dispatch) {
           type: ActionTypes.SET_PREFERENCES,
           preferences: response.data.preferences
         });
+        const valorLanguage = response.data.preferences.language;
```

Comment:
What does `valorLanguage` mean?

Reply:
Hi, it is response.data.preferences.language , I forgot to use the same way to access it in both parts

Reply:
But I meant, what does the word `valor` mean as a variable name?

Reply:
it means value, so a proper name for this intermediate variable would be languageValue, not this thing, I will get rid of this.

Reply:
@andrewn I've deleted the variable.